### PR TITLE
Add script to check SetMediaClockTimer with _wrong json syntax in ena…

### PR DESCRIPTION
**Pull request by OVikhrov**
…bleSeek
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Json_Syntax_In_Parameter.lua [hunk 1]
```diff
@@ -0,0 +1,104 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Json_Syntax_In_Parameter.lua]: @OVikhrov, please update  Requirement summary

File: test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Json_Syntax_In_Parameter.lua [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+-- INVALID_DATA
+--
+-- Description:
+-- In case the request comes to SDL with wrong json syntax, SDL must respond with resultCode "INVALID_DATA" and success:"false" value.
+--
+-- 1. Used preconditions:
+-- a) First SDL life cycle
+-- b) SetMediaClockTimer RPC allowed in preloaded file for default app
+-- c) App successfylly registered, consented and activated
```
Comment (aarotar) [inline test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Json_Syntax_In_Parameter.lua]: @OVikhrov, please check syntax in script

File: test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Json_Syntax_In_Parameter.lua [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_SetMediaClockTimer_With_Wrong_Json_Syntax_in_enableSeek()
+self.mobileSession.correlationId = self.mobileSession.correlationId + 1
+
```
Comment (aarotar) [inline test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Json_Syntax_In_Parameter.lua]: @OVikhrov, plase avoid redundant empty lines and use lua beautifier

**Comment by OVikhrov** [comment]
Accepted. Corrected in 7a7c159ed3298f17739ccd46d066753b0968c92c

File: test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Json_Syntax_In_Parameter.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/SetMediaClockTimer/ATF_N_SetMediaClockTimer_enableSeek_Wrong_Json_Syntax_In_Parameter.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**